modules/passkey/contracts/base/SignatureValidator.sol

```diff
@@ -28,7 +38,17 @@ abstract contract SignatureValidator {
      * @return magicValue The magic value indicating the validity of the signature.
      */
     function isValidSignature(bytes32 message, bytes calldata signature) external view returns (bytes4 magicValue) {
-        if (_verifySignature(message, signature)) {
+        uint256 x;
+        uint256 y;
+        P256.Verifiers verifiers;
+        // solhint-disable-next-line no-inline-assembly
+        assembly {
+            x := calldataload(sub(calldatasize(), 88))
+            y := calldataload(sub(calldatasize(), 56))
+            verifiers := shr(64, calldataload(sub(calldatasize(), 24)))
+        }
```

Comment:
Hmm, to me this feels like it should be part of `_verifySignature` as it is duplicated and not part of the base `SignatureValidator`.

The `SignatureValidator` base class is a *general* implementation of a smart contract owner for Safes, and the fact that we are using this trusted forwarder patterns is an implementation detail of the actual signer, and not of the base class.

Reply:
https://github.com/safe-global/safe-modules/pull/370/commits/954078b6179e8fea2c8b7da9774a779281d2428d